src/System.Runtime/tests/System/Reflection/AssemblyNameTests.cs

```diff
@@ -75,6 +76,7 @@ public static void Clone()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot, "AssemblyName.GetAssemblyName() not supported on UapAot")]
```

Comment:
There no plans to support any of these that are marked "PNSE" or "not supported on UapAot."